reactnative/RNPlayground/app/screens/AboutScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { StackScreenProps } from "@react-navigation/stack"
+import { AppStackScreenProps } from "../navigators"
+import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import {
+  Text, Screen
+} from "../components"
+import { isRTL } from "../i18n"
+import { colors, spacing } from "../theme"
+import { useSafeAreaInsetsStyle } from "../utils/useSafeAreaInsetsStyle"
+
+export const AboutScreen: FC<StackScreenProps<AppStackScreenProps, "About">> = observer(function AboutScreen() {
+    const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"])
```

Comment:
this notation, `$bottomContainerInsets` what's the reasoning behind it? local variables you make them be prefix with `$` or what?

Reply:
Purely depends on project convention, I saw some examples online that used the `$camelCase` for `View` components - but unfortunately React Native has no official naming conventions.

Happy to change if needed. I agree it may get confusing with template literals, e.g. `string text ${ expression } `.

Might also be good next steps to have a discussion or setup a linting tool, so project conventions we establish will be applied consistently.